src/components/ClassCard/index.stories.js

```diff
@@ -1,54 +0,0 @@
-import React from 'react'
```

Comment:
ClassCard will still be available for those that need it or are currently using it.  But so that we opt for context-less components, I am removing it from the documentation.